ports/bsd/rs485.c

```diff
@@ -0,0 +1,881 @@
+/**
+ * @file port/bsd/rs485.c
+ * @brief Provides BSD/Darwin(macOS) specific functions for RS-485 serial
+ * operation
+ * @author Steve Karg <skarg@users.sourceforge.net>
+ * @author Nikola Jelic <nikola.jelic@euroicc.com>
+ * @author Michael O'Neill <em.pee.oh@gmail.com>
+ * @date 2007, 2011, 2024
+ *
+ * @copyright SPDX-License-Identifier: MIT
+ */
+
+/* The module handles sending data out the RS-485 port */
```

Comment:
no need to have these duplicitous comments, please remove

Reply:
duplicative or duplicitous? If you mean the latter, yes Line 13 is definitely redundant. But if you meant the former, I meant no offense and I can remove lines 7 and 6 and edit the dates on 8.

Reply:
redundant - no offense taken. Less lines means less code to maintain.